src/mame/konami/konami_gn676_lan.h

```diff
@@ -1,42 +1,72 @@
 // license:BSD-3-Clause
-// copyright-holders:Ville Linde
+// copyright-holders:Ville Linde, windyfairy
 
-#ifndef MAME_MACHINE_KONAMI_GN676_LAN_H
-#define MAME_MACHINE_KONAMI_GN676_LAN_H
+#ifndef MAME_KONAMI_GN676_LAN_H
+#define MAME_KONAMI_GN676_LAN_H
 
 #pragma once
 
-class konami_gn676_lan_device :  public device_t
+#include "machine/x76f041.h"
+
+class konami_gn676_lan_device : public device_t
 {
 public:
```

Comment:
Please set the unemulated LAN flag on this device (you do it by implementing `unemulated_features` – grep for a device that does it) so it will be picked up by all systems using the device automatically.  Then the systems don’t need the flag themselves if they don’t have any other unemulated LAN hardware.